lib/src/response.dart

```diff
@@ -25,7 +26,18 @@ class Response extends BaseResponse {
   /// [RFC 2616][].
   ///
   /// [RFC 2616]: http://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html
-  String get body => _encodingForHeaders(headers).decode(bodyBytes);
+  String parseBody() {
```

Comment:
This seems unrelated to the title of the PR

Reply:
Your'e correct. I use my fork for personal projects and forgot this PR was still open. I am going to close and open a new one with just the follow redirects only.

Reply:
I'd do seperate PRs for protli and proxy support